### DISCUSSION TITLE
How can I avoid multiple notifications for a crash loop? I'm receiving logs multiple times to slack

### DISCUSSION THREAD

**Discussion by shameemshah** [abahmed/kwatch#287] [Q&A]
How can I avoid multiple notifications for a crash loop? I'm receiving logs multiple times

**Comment by abahmed** [comment]
Hi @shameemshah — we’ve been working on significant enhancements and improvements to kwatch, including correlation, grouping, and cooldown handling for crash loops. These changes have not been released yet.

The upcoming version is designed to notify once for the initial failure, then group and suppress repeated events for the same incident during the cooldown period.

We’ll share an update once the improved version is released. Sorry for the delay, and thank you for your patience.
